.github/workflows/benchmark_torch_compile_nightly.yml

```diff
@@ -20,10 +20,10 @@ jobs:
           cd ..
           pwd
           rm -rf _tool
-      - name: Setup Python 3.8
+      - name: Setup Python 3.10
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.10.9
```

Comment:
can we just have 3.10?

Reply:
One of the previous runs had failed when I specified just 3.10: https://github.com/pytorch/serve/actions/runs/7255048620/job/19764929373

Commit ID: https://github.com/pytorch/serve/pull/2856/commits/bcda4791a30027ac5a1ce13665b5fdd87b335568

We can choose the versions according to https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json.

Reply:
You need to use quotes, check this workflow https://github.com/pytorch/serve/blob/master/.github/workflows/regression_tests_cpu_binaries.yml#L20

Reply:
Oh I understand. I will update the workflow accordingly.